bin/bucc

```diff
@@ -321,22 +321,35 @@ _fly() {
         popd
     fi
 
-    faketty fly --target bucc login \
+    fly_target=bucc
```

Comment:
if the default is already bucc why do we put in the default vars files?

Reply:
The idea was to show ppl in their vars.yml how to override it. But some existing users of bucc won’t have it in their vars.yml so need the default option too.